src/doc/rustc/src/platform-support/armv7a-vex-v5.md

```diff
@@ -0,0 +1,107 @@
+# `armv7a-vex-v5`
+
+**Tier: 3**
+
+Allows compiling user programs for the [VEX V5 Brain](https://www.vexrobotics.com/276-4810.html), a microcontroller for educational and competitive robotics.
+
+Rust support for this target is not affiliated with VEX Robotics or IFI.
+
+## Target maintainers
+
+This target is maintained by members of the [vexide](https://github.com/vexide) organization:
+
+- [@max-niederman](https://github.com/max-niederman)
+- [@Tropix126](https://github.com/Tropix126)
+- [@Gavin-Niederman](https://github.com/Gavin-Niederman)
+- [@doinkythederp](https://github.com/doinkythederp)
+
+## Requirements
+
+This target is cross-compiled. Dynamic linking is unsupported.
+
+`#![no_std]` crates can be built using `build-std` to build `core` and optionally
+`alloc`. Unwinding panics are not yet supported. `std` is partially implemented, see below.
+
+This target generates binaries in the ELF format that may uploaded to the brain with external tools.
+
+## Standard Library Support
+
+`armv7a-vex-v5` implements as much of the standard library as is possible using only public VEX SDK functions.
+This includes:
+
+- `std::time`, not including `SystemTime` as the SDK does not provide absolute time information.
+- `std::io`, including `stdin()` and `stdout()`, but not `stderr()`, as stderr does not exist on this platform.
```

Comment:
Stderr should be changed to point to the same sink as stdout so error output is not lost. It's fine to have both stdout and stderr output to the same sink.